dialogue-example/dialogue-example-dialogue/build/reports/checkstyle/main.html

```diff
@@ -0,0 +1,119 @@
+<html>
```

Comment:
I think we're missing `build/` in the .gitignore

Reply:
🤦‍♂